templates/eventstore/0/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '2'
+services:
+  eventstore:
+    image: eventstore/eventstore
+    entrypoint: /bin/bash
+    command:
+    - -c
+    - '
+      apt-get install curl &&
+      export HOST_PUBLIC_IP=$$(curl http://rancher-metadata/2015-12-19/self/host/agent_ip) &&
+      grep -q -F "IntIpAdvertiseAs: $$(hostname -i)" /etc/eventstore/eventstore.conf  ||
+      echo "IntIpAdvertiseAs: $$(hostname -i)" >> /etc/eventstore/eventstore.conf &&
+      grep -q -F "ExtIpAdvertiseAs: $$HOST_PUBLIC_IP" /etc/eventstore/eventstore.conf  ||
+      echo "ExtIpAdvertiseAs: $$HOST_PUBLIC_IP" >> /etc/eventstore/eventstore.conf &&
+      /entrypoint.sh'
+
+    environment:
+    - EVENTSTORE_CLUSTER_SIZE=3
+    - EVENTSTORE_CLUSTER_DNS=eventstore
+    - EVENTSTORE_CLUSTER_GOSSIP_PORT=${internal_http_port}
+    - EVENTSTORE_INT_HTTP_PREFIXES=http://*:${internal_http_port}/
+    - EVENTSTORE_EXT_HTTP_PREFIXES=http://*:${external_http_port}/
+    - EVENTSTORE_INT_HTTP_PORT=${internal_http_port}
+    - EVENTSTORE_INT_TCP_PORT=${internal_tcp_port}
+    - EVENTSTORE_EXT_HTTP_PORT=${external_http_port}
+    - EVENTSTORE_EXT_TCP_PORT=${external_tcp_port}
+    
+    ports:
```

Comment:
Not needed if you want to use load balancer. 
Could you add lb config??

Reply:
It's better if the eventstore client have direct access to each of the nodes. So I gave up having a LB.